cmd/entry.go

```diff
@@ -1,7 +1,16 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+	"go-calendar-practice/pkg/loaders"
+
+	"github.com/gin-gonic/gin"
+)
 
 func main() {
-	fmt.Printf("Hello World!")
+	r := gin.Default()
```

Comment:
```suggestion
	router := gin.Default()
```
약어 사용은 최대한 풀어주세요

Go에서는 같은 변수명은 재선언 등이 아예 안되서, 짧은 범위에서도 줄여쓰는 것보다 명시적으로 구분해주는 편이 좋습니다

보통 한글자 축약어는 데코레이터 패턴에서 (아주 권장되지는 않지만)한정적으로 사용하곤 합니다.

Reply:
아하! 저도 약어는 최대한 풀어서 쓰는 걸 지향하는데 gin 라이브러리의 예제 코드 스타일이 router를 r로 표현하는 거 같아 일단 따랐었습니다..! 그게 권장사항이 아니였군요
반영하도록 하겠습니다

Reply:
[공식 Gin Repository](https://github.com/gin-gonic/gin) 에서 API Example을 확인해보면 router로 적는게 표준이긴 합니다!

아마 QuickStart 예제를 바탕으로 초기 Framework 구축하는 부분에서 다들 r로 적는거같아요